### DISCUSSION TITLE
Notify 1.4.1 doesn't show anything after update

### DISCUSSION THREAD

**Discussion by serrq** [aritra-tech/Notify#124] [General]
Import a backup not even run!
What does it happens?

![1000042122](https://github.com/aritra-tech/Notify/assets/113377431/90a12c2a-84bb-43e3-8184-d3ef8ebb2dd5)

**Comment by serrq** [comment]
Full internet access again appeared.


1.3.1
![1000043322](https://github.com/aritra-tech/Notify/assets/113377431/74c9aa48-0157-4cca-9aba-2b1f2be2c6e6)

1.4.1
![1000043321](https://github.com/aritra-tech/Notify/assets/113377431/667be70d-2dc7-4988-973a-a0f549816fc2)

**Comment by serrq** [comment]
Found the problem!

Previously generated backup is broken or incompatible. It has data inside, but doesn't work when imported. So I cannot more accessing to my old notes.

**Reply by aritra-tech** [reply]
Sorry for the loss of your notes.
I will be fixing this issue as soon as possible.
Sorry for the inconvenience.

**Comment by serrq** [comment]
In next update will I use again my old backup?

**Reply by aritra-tech** [reply]
How do you update Notify through play store?

**Reply by serrq** [reply]
Through Github.

**Reply by aritra-tech** [reply]
I prefer to download or update it through the Play Store.
When I release the latest update on GitHub at that time only I upload the updated version on Play Store it takes time & you don't have to delete your previous notes.

**Reply by serrq** [reply]
I didn’t delete them. Notify has deleted them when I switched from version 1.3.1 to 1.4.1. At that point I had the "Notes" tab empty. 

It also does not load the backup, so at the moment it is impossible to perform a restore.

**Reply by serrq** [reply]
1.4.1 has led the "full internet" access again. A mistake or you changed your mind about it?

**Reply by aritra-tech** [reply]
Internet access is needed because I have implemented "In-App Update" that's why it needs internet access.
I hope you understand.

**Comment by serrq** [comment]
1.5.1 I cannot restore nothing. Backup restoring doesn't work on my phone. Is it a permission problem? Did you forget to add some permission request like write/read local storage or something else?

**Comment by serrq** [comment]
Okay for internet permission (fdroid version of Notify doesn't need of it), but the anti-features in 1.5.1? Might you solve this in next version?

![1000093056](https://github.com/aritra-tech/Notify/assets/113377431/8df79605-e68d-4986-b415-64e105663373)
